### PR TITLE
Decaff driver (final)

### PR DESCRIPTION
**Pull request by sainthkh**
Part of #2690

Final decaff of the `driver` package.

# NEVER SQUASH THIS PR.

### CODE REVIEW

**Comment by sainthkh** [comment]
It seems that flaky test happened.

File: packages/driver/src/cypress/error_messages.js [hunk 1]
```diff
@@ -0,0 +1,1795 @@
...
+
+const getHttpProps = (fields = []) => {
+  return _
+  .chain(fields)
+  .reduce(formatProp, [])
+  .join('\n')
+  .value()
+}
+
+const cyStripIndent = (str, indentSize) => {
```
Comment (jennifer-shehane) [inline packages/driver/src/cypress/error_messages.js:85]: Can you add a comment here explaining why someone would need to use function this over the included `stripIndent` function?

File: packages/server/__snapshots__/4_request_spec.coffee.js [hunk 2]
```diff
@@ -374,6 +373,7 @@ Body: <!DOCTYPE html>
 </body>
 </html>
 
+
```
[the remark this answers is not in the document]
Comment (sainthkh) [reply packages/server/__snapshots__/4_request_spec.coffee.js]: This problem needs a research. 

When I reverted the change and run the tests on my local Ubuntu, it works. But when I pushed it, it fails.

[the remark this answers is not in the document]
Comment (sainthkh) [reply packages/server/__snapshots__/4_request_spec.coffee.js]: I still don't know why it happened. But the fix was easy.

**Comment by sainthkh** [comment]
Flaky failures.

**Comment by sainthkh** [comment]
Another flaky failure.

**Comment by jennifer-shehane** [comment]
I think we may need to look closer at these failures. They are failing within some tests that were decaffeinated, although I don't understand why that would affect anything. But, I've rerun these a lot and they keep failing.

Screen Shot 2020-05-26 at 8 05 40 PM

**Comment by sainthkh** [comment]
Fixed conflicts and rebased. Interestingly, socket fail didn't happen!

**Comment by sainthkh** [comment]
@jennifer-shehane Unfortunately, there are a lot of coffeescript codes in `packages/server`. We can remove coffeescript-related tools after they're converted.

**Pull request merged**